tickets/DM-25174_zogy_tn/zogy_notes.tex

```diff
@@ -157,11 +158,13 @@ \section{The theoretical solution of the ZOGY matching kernel and difference
 factors are unity for simplicity. As the input PSFs are pure Gaussians,
 i.e.\ symmetric, real value functions, their Fourier transforms are also
 real and symmetric.\footnote{Detailed calculation notebooks are part of
-  DM-26087.} PSF1 i Note the different behavior of the two matching kernels
-towards high frequencies. The matching kernel of the narrower PSF image
-\(c_1\) goes to zero, while the other one goes to unity here. While the
-graphs of \(c_1\) and \(c_2\) resemble to Gaussians, they are not any
-more. We calculate their inverse Fourier transform for points along the x-axis by numerical integration.
+  DM-26087.} Note the different behavior of the two matching kernels towards
+high frequencies. The matching kernel of the narrower PSF image \(c_1\) goes
+to zero, while the other one goes to unity here. While the graphs of \(c_1\)
+and \(c_2\) resemble to Gaussians, they are not any more, and we must use
+numerical integration to calculate their inverse Fourier transform. Their
+image space values for points along the x-axis is shown in
+\Cref{fig:theo_Gaussians_img_c1,fig:theo_Gaussians_img_c2}.
 %
 \begin{figure}
 \begin{center}
```

Comment:
There's an odd kink at the origin in the plot of psf1 in the PDF. Please check the input data and see if you can smooth that out, perhaps with higher resolution